doc/start/hardware-recommendations.rst

```diff
@@ -255,12 +268,19 @@ cabling more manageable. VLANs using 802.1q protocol require VLAN-capable NICs
 and Switches. The added hardware expense may be offset by the operational cost
 savings for network setup and maintenance. When using VLANs to handle VM
 traffic between the cluster and compute stacks (e.g., OpenStack, CloudStack,
-etc.), it is also worth considering using 10G Ethernet. Top-of-rack routers for
-each network also need to be able to communicate with spine routers that have
-even faster throughput--e.g.,  40Gbps to 100Gbps.
+etc.), there is additional value in using 10G Ethernet or better; 40Gb or
+25/50/100 Gb networking as of 2020 is common for production clusters.
+
+Top-of-rack routers for each network also need to be able to communicate with
+spine routers that have even faster throughput--e.g.,  40Gbps to 100Gbps.
```

Comment:
I don't think that the comma is necessary after "e.g".

Reply:
Rephrased to obviate.